containers/README.md

```diff
@@ -4,3 +4,21 @@ Container image gets automatically built by quay.io at [Cerberus image](https://
 
 ### Run containerized version
 Refer to the [instructions](https://github.com/openshift-scale/cerberus#Run-containerized-version) for information on how to build and run the containerized version of cerberus.
+
+### Cerberus as a KubeApp
+To run containerized Cerberus as a Kubernetes/OpenShift Deployment, follow these steps:
+1. Configure the [config.yaml](https://github.com/openshift-scale/cerberus/tree/master/config) file according to your requirements.
+2. Create a namespace under which you want to run the cerberus pod using `kubectl create ns <namespace>`.
+3. Switch to `<namespace>` namespace:
+    - In Kubernetes, use `kubectl config set-context --current --namespace=<namespace>`
+    - In OpenShift, use `oc project <namespace>`
+4. Create a ConfigMap named kube-config using `kubectl create configmap kube-config --from-file=<path_to_kubeconfig>`
+5. Create a ConfigMap named cerberus-config using `kubectl create configmap cerberus-config --from-file=<path_to_cerberus_config>`
+6. Create a serviceaccount to run the cerberus pod with privileges using `kubectl create serviceaccount useroot`.
+    - In Openshift, execute `oc adm policy add-scc-to-user privileged -z useroot`.
+7. Create a Deployment and a NodePort Service using `kubectl apply -f cerberus.yml`
+8. Accessing the go/no-go signal:
+    - In Kubernetes, execute `kubectl port-forward --address 0.0.0.0 pod/<cerberus_pod_name> 8080:8080` and access the signal at http://localhost:8080.
```

Comment:
Maybe also mention that the signal can be accessed at public ip/hostname on port 8080?